src/pages/login/Login.tsx

```diff
@@ -1,7 +1,113 @@
-import { FC } from "react";
+import { FC, useState } from "react";
+import { Link } from "react-router-dom";
+
+//import photos
+import vector from "../../assets/photos/vector-login.svg";
+import logo from "../../assets/logos/icon-transparent.svg";
+
+//import icons
+import {
+  AiOutlineArrowLeft,
+  AiFillEye,
+  AiFillEyeInvisible,
+} from "react-icons/ai";
 
 const Login: FC = () => {
-	return <div>Login</div>;
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [showPassword, setShowPassword] = useState<boolean>(false);
+
+  //for submit by clicking an enter key
+  const handleKeyPress = (e: any) => {
+    if (e.key === "Enter") {
+      if (email === "" || password === "") {
+        return;
+      } else {
+        console.log({
+          Email: email,
+          Password: password,
+        });
+      }
+    }
+  };
```

Comment:
you can remove `handleKeyPress` because form's `onSubmit` event  triggers on both `enter` and `click`